ClassPaulita/GuessTheCardGame.cs

```diff
@@ -10,37 +10,30 @@ public class GuessTheCardGame
         private List<KeyValuePair<int, string>> _scoreBoard = new List<KeyValuePair<int, string>>();
         private Random _random = new Random();
         private string _spacing = "\n\t\t\t\t\t\t";
-        
 
         public GuessTheCardGame()
         {
-
             if (File.Exists(_fileName))
             {
                 StreamReader reader = File.OpenText(_fileName);
 
-                while (true)
+                string line = reader.ReadLine();
+
+                while (line != null)
                 {
-                    string line = reader.ReadLine();
+                    string[] scoreAndName = line.Split(' ');
 
-                    if (line == null)
-                    {
-                        break;
-                    }
-                    else
+                    if (scoreAndName.Length == 2)
                     {
-                        string[] scoreAndName = line.Split(' ');
+                        bool isItConvertible = int.TryParse(scoreAndName[0], out int result);
 
-                        if (scoreAndName.Length == 2)
+                        if (isItConvertible && scoreAndName[1] != null)
```

Comment:
I like that you double checked, but it would be better to check both if it is null, empty, or whitespace.

![image](https://user-images.githubusercontent.com/43797367/61414569-48310180-a8bc-11e9-979a-dce1bc200ed2.png)

Reply:
what do you mean?

Reply:
I fixed the issue. Thanks!